roles/telegraf/defaults/main.yml

```diff
@@ -13,3 +13,7 @@ telegraf_directories:
 
 telegraf_config_file_path: "{{ telegraf_config_dir }}/{{ telegraf_config_file_name }}"
 telegraf_log_file_path: "{{ telegraf_log_dir }}/{{ service_identifier }}.log"
+
+telegraf_supra_log_file_path:
```

Comment:
why do we need two different variables for validator and fullnode here ? looks like both have /opt/{{ node_service_identifier }}